### PR TITLE
Adding starter code and edge worker working solution

### PR DESCRIPTION
**Pull request by gannaraputeja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Uniform Akamai EdgeWorker
+
+See docs [here](https://docs.levo.ai/).
+
+## Getting Started
+1. Ensure you have [Akamai CLI installed](https://techdocs.akamai.com/edgeworkers/docs/akamai-cli)
+    - On Windows you may need to add `akamai.exe` location to `Path` environment variable
+2. Ensure you have [Akamai API credentials](https://techdocs.akamai.com/developer/docs/set-up-authentication-credentials) file
+3. Install edgeworkers command with `akamai install edgeworkers`
+4. Ensure you have [EdgeWorker](https://control.akamai.com/apps/edgeworkers/#/edgeworkers) created
```
Comment (buchi-busireddy) [inline README.md:10]: @gannaraputeja don't we need to give the command to create the edge worker first?

File: src/bundle.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "edgeworker-version": "1.29",
```
Comment (buchi-busireddy) [inline src/bundle.json]: Should we start with `0.1`?

File: src/bundle.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "description": "Levo Edge Worker 1.29",
+  "config": {
+    "logging": {
+      "level": "debug"
+    },
+    "subrequest": {
+      "invoke-for-edgeworker-clients": true
```
Comment (buchi-busireddy) [inline src/bundle.json:9]: Please add comments on why

File: src/main.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    try {
+      const logResponse = await httpRequest(`${logEndpoint}`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(logData)
+      });
+
+      logger.info('Log data sent. Status: ' + logResponse.status);
```
Comment (buchi-busireddy) [inline src/main.js]: Convert into debug?

File: src/main.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    } catch (error) {
+      logger.error('Error sending log data: ' + error.toString());
```
Comment (buchi-busireddy) [inline src/main.js]: Could this flood their logs or something if we don't limit it? What's the best practice here?

File: src/main.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    return response;
+  } catch (error) {
+    logger.error('Error in onClientResponse: ' + error.toString());
```
Comment (buchi-busireddy) [inline src/main.js]: I don't think we should ever modify the response right?

File: src/bundle.json (hunk 2 above)
Comment (gannaraputeja) [reply src/bundle.json]: Updated to 0.1.0

File: src/bundle.json (hunk 3 above)
Comment (gannaraputeja) [reply src/bundle.json:9]: We dont require this anymore. removed

File: src/main.js (hunk 4 above)
Comment (gannaraputeja) [reply src/main.js]: Addressed

File: src/main.js (hunk 5 above)
Comment (gannaraputeja) [reply src/main.js]: Added rate limiting with default 5 error logs per minute

File: src/main.js (hunk 6 above)
Comment (gannaraputeja) [reply src/main.js]: Yes. Updated to return response object.

File: README.md (hunk 1 above)
Comment (gannaraputeja) [reply README.md:10]: I don't see command to create edgeworker via cli. https://github.com/akamai/cli-edgeworkers
It is only supporting to register/update edge worker. Creation is through UI only.

Comment (buchi-busireddy) [reply README.md:10]: Do we need screenshots to create the edge worker?

File: src/bundle.json (hunk 3 above)
Comment (gannaraputeja) [reply src/bundle.json:9]: Looks like we need this to make request to edge URL echo.getlevoai.com or akamai edge collector url

File: src/main.js (hunk 5 above)
Comment (gannaraputeja) [reply src/main.js]: Changed this to use sampling

**Review by bumblebee338 — APPROVED** [review]
LGTM

**Pull request merged**